client/components/Agendas/AgendaListItem.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import moment from 'moment';
+import {IAgendaEntity} from '../../interfaces';
+import {IPropsGenericFormItemComponent} from 'superdesk-api';
+import {ListItem, ListItemColumn, ListItemActionsMenu} from 'superdesk-core/scripts/core/components/ListItem';
+import {getFormFieldPreviewComponent} from 'superdesk-core/scripts/core/ui/components/generic-form/form-field';
+import {gettext} from '../../utils';
+import {getNameField} from './ManageAgendasModal';
+import {IconButton, Label} from 'superdesk-ui-framework';
```

Comment:
do we still need to import from 'superdesk-ui-framework/react'?
```suggestion
import {IconButton, Label} from 'superdesk-ui-framework/react';
```

Reply:
yes, I just keep forgetting it since it's manual. In some cases the app just crashes at runtime if we don't so I always try to do it

Reply:
so we should import from `superdesk-ui-framework/react` correct?